_errors.md

```diff
@@ -4,13 +4,13 @@ Uphold API uses the following error codes:
 
 Code | Meaning
 ---- | ---------------------------------------------------------------------------------------------------
-400  | Bad Request -- Validation failed.
-401  | Unauthorized -- Bad credentials.
-403  | Forbidden -- Access forbidden.
-404  | Not Found -- Object not found.
-409  | Conflict -- Entity already exists.
-412  | Precondition Failed
-416  | Requested Range Not Satisfiable
-429  | Too Many Requests -- Rate limit exceeded.
-500  | Internal Server Error -- Something went wrong in our server.
-503  | Service Unavailable -- We're temporarily offline for maintenance. Please try again in a little bit.
+400  | [Bad Request](https://tools.ietf.org/html/rfc7231#section-6.5.1) — Validation failed.
+401  | [Unauthorized](https://tools.ietf.org/html/rfc7235#section-3.1) — Bad credentials.
+403  | [Forbidden](https://tools.ietf.org/html/rfc7231#section-6.5.3) — Access forbidden.
+404  | [Not Found](https://tools.ietf.org/html/rfc7231#section-6.5.4) — Object not found.
+409  | [Conflict](https://tools.ietf.org/html/rfc7231#section-6.5.8) — Entity already exists.
+412  | [Precondition Failed](https://tools.ietf.org/html/rfc7232#section-4.2)
```

Comment:
This is only mentioned in the docs regarding a malformed `Range` header (in the [Pagination section](https://github.com/uphold/docs/blob/master/_pagination.md)). Is it OK to add that description here, or do we use this error in other situation as well?

In any case, it seems odd for this to be the only error in this list without a custom description related to our API.

Reply:
I don't think the description makes sense here as it is endpoint-specific. A 400 Bad Request may not always be a Validation failed error. 

We're missing a 416 here as well.

Reply:
Good point. In fact, the descriptions we have here are essentially made obsolete by adding the links, which provide the canonical descriptions for these errors. What we have here is mostly redundant information, and in the cases it's not, it's inappropriately specific, as you point out. I'll go ahead and remove our custom descriptions.

> We're missing a 416 here as well.

Not sure what you mean — isn't that in the line immediately below?

Reply:
I think I've missed the first time, sorry!